### PR TITLE
add template for norbornane

### PR DESCRIPTION
**Pull request by ZontaNicola**
added template, also modified the ring detection algorithm so that norbornane triggers the template search

### CODE REVIEW

File: CoordgenFragmentBuilder.cpp [hunk 1]
```diff
@@ -1045,6 +1045,12 @@ void CoordgenFragmentBuilder::simplifyRingSystem(
                         /* don't separate rings of bicyclo (1,1,1) pentane so we can use a template instead */
                         n++;
                     }
+                    if (r->fusionAtoms.at(ringCounter).size() == 3 &&
+                        r->size() == 5 &&
+                        fusedRing->size() == 5) {
```
Comment (d-b-w) [inline CoordgenFragmentBuilder.cpp:1050]: What's the cost of this?

Like, if we don't separate the rings, but it isn't actually norborane... Would that make the generated structure worse? Or would that break matching to some other templates?

**Comment by d-b-w** [comment]
Nic -
I noticed that QA closed https://jira.schrodinger.com/browse/CRDGEN-274 as fixed. Is this still needed?

File: CoordgenFragmentBuilder.cpp (hunk 1 above)
Comment (ZontaNicola) [reply CoordgenFragmentBuilder.cpp:1050]: I'm not sure there's another way to fuse two pentagons on 3 consecutive vertices other than the structure of norbornane... but if we don't separate here and don't find a template then the structure will be generated with the "as planar as possible" approach, which should still do a pretty good job here

**Comment by ZontaNicola** [comment]
> Nic -
> I noticed that QA closed https://jira.schrodinger.com/browse/CRDGEN-274 as fixed. Is this still needed?

oh yeah! looks like we're doing well without a template... maybe I'd still merge this so we get the nice hexagonal shape with the template but the other one honestly looks really nice too

**Comment by ZontaNicola** [comment]
I don't think this is needed anymore. Closing the PR

**Pull request closed without merging**